hwdb/parse_hwdb.py

```diff
@@ -112,6 +112,8 @@ def property_grammar():
              ('ID_INPUT_TOUCHPAD_INTEGRATION', Or(('internal', 'external'))),
              ('XKB_FIXED_LAYOUT', STRING),
              ('XKB_FIXED_VARIANT', STRING),
+             ('KEYBOARD_LED_NUMLOCK', Literal('0')),
+             ('KEYBOARD_LED_CAPSLOCK', Literal('0')),
```

Comment:
I don't think that's super useful, when we say that presence is implicit. It *could* be used to override an already existing absence, but I'm not even sure whether hwdb matches all the possible matches, or does so in order.

We might also want to use proper strings (`present` and `absent`). Really no preference here.